modules/igv/1.0/etc/liftover_regions.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
This did not work? https://github.com/LCR-BCCRC/lcr-scripts/blob/master/crossmap/1.1/convert_maf_coords.sh
If this script has something specific the existing lcr-script doesn't support already, should we instead add that as 1.2 version instead of creating new one? 😄

Reply:
I switched to using liftover and moved the main functions of the liftover_regions.sh script to the actual rule itself